activestorage/test/models/blob/analyzable_test.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "database/setup"
+
+class ActiveStorage::Blob::AnalyzableTest < ActiveSupport::TestCase
+  class AnalyzerA < ActiveStorage::Analyzer
+    def self.accept?(blob)
+      true
+    end
+
+    def metadata
+      { analyzed_by: { a: true } }
+    end
+  end
+
+  class AnalyzerB < ActiveStorage::Analyzer
+    def self.accept?(blob)
+      true
+    end
+
+    def metadata
+      { analyzed_by: { b: true } }
+    end
+  end
+
+  class AnalyzerC < ActiveStorage::Analyzer
+    def self.accept?(blob)
+      false
+    end
+
+    def metadata
+      { analyzed_by: { c: true } }
+    end
+  end
+
+  test "#analyze supports multiple analyzers for a Blob" do
```

Comment:
Note to self: add another test to ensure that analyze is set to false if all analyzers return false for accept?